samples/hssi/hssi_10g_cmd.h

```diff
@@ -174,10 +174,15 @@ class hssi_10g_cmd : public hssi_cmd
               << "  eth: " << eth_ifc << std::endl
               << std::endl;
 
-    if (eth_loopback_ == "on")
-      enable_eth_loopback(eth_ifc, true);
-    else
-      enable_eth_loopback(eth_ifc, false);
+    if (eth_ifc == "") {
```

Comment:
Enable loopback is not valid if eht_ifc is empty

eth_loopback_  is not valid if eht_ifc is empty